main.cpp

```diff
@@ -28,510 +28,592 @@ Thing: Car Wash
     {
-        // 5 properties
-//      1) number of functioning eyes
         int numberOfFunctioningEyes = 2;
-//      2) vision quality
         double visionQuality = 65.5;
-//      3) blinks per second
         float blinksPerSecond = 10.0f;
-//      4) year of last eye test
         std::string yearLastEyeTest = "1985";
-//      5) Peripheral vision in degrees
         int peripheralVision = 165;
 
-        // 3 actions
-//    1) check vision
-        void checkVision(double visionQuality);
-//    2) find probability of seeing object
-        Vision probabilityOfSighting(Vision vision);
-//    3) get glasses prescription
-        Vision getGlasseesPrescription(Vision vision);        
+        void checkVision(double eyesightQuality);
+        void probabilityOfSighting(bool morning);
+        void getGlassesPrescription();
     };
 
-//    1) number of neurons
-// can I use long long?
     double numNeurons = 10e13;
-//    2) number of functioning hemispheres
     int numWorkingHemispheres = 2;
-//    3) memory storage in gigabytes
     float storageSpaceInGigabytes = 1000000.0f;
-//    4) calculations per second
     double calculationsPerSecond = 10e15;
-//    5) unknown regions
     float percentageOfUnknownBrainRegions = 0;
 
-//    1) recognise faces
     void recogniseFace(bool awake, int numberMeetingsWithPerson);
-//    2) estimate distances
-    void estimateDistance(Vision vision);
-//    3) construct a language
+    void estimateDistance(int distance);
     void constructLanguage(int age);
 };
 
-/*
-Thing 4) chord
-5 properties:
-    1) number of notes
-    2) possible next chords within key
-    3) chord number in roman numerals
-    4) has chord been altered
-    5) possible preceding chords within key
-3 things it can do:
-    1) make a sound
-    2) be played as arpeggio
-    3) be omitted
- */
+void Brain::recogniseFace(bool awake, int numberMeetingsWithPerson)
+{
+    if (awake && numberMeetingsWithPerson >= 2)
+    {
+        std::cout << "Face recognised" << std::endl;
+    }
+    else
+    {
+        std::cout << "Face not recognised" << std::endl;
+    }
 
-struct Chord
+}
+
+void Brain::estimateDistance(int distance)
 {
+    if (distance < 250)
+    {
+        std::cout << "Distance estimated" << std::endl;
+    }
+    else
+    {
+        std::cout << "Distance could not be estimated" << std::endl;
+    }
+
+
+}
 
-//    1) number of notes
+void Brain::constructLanguage(int age)
+{
+    bool languageSuccess;
+    if (age >= 2)
+    {
+        languageSuccess = true;
+    }
+    else
+    {
+        languageSuccess = false;
+    }
+}
+void Brain::Vision::checkVision(double eyesightQuality)
+{
+    if (eyesightQuality < 45.8)
+    {
+        getGlassesPrescription();
+    }
+}
+
+void Brain::Vision::probabilityOfSighting(bool morning)
+{
+    if (morning)
+    {
+        std::cout << " >=50% Chance of sighting" << std::endl;
+    }
+    else
+    {
+        std::cout << "<50% chance of sighting" << std::endl;
+    }
```

Comment:
add space around your operators, even if they're in quotes.  it improves the readability of your code
- [ ] revise

Reply:
Done.